src/MySqlConnector/project.json

```diff
@@ -37,6 +37,7 @@
 				"System.Globalization": "4.0.11-*",
 				"System.IO": "4.1.0-*",
 				"System.Linq": "4.1.0-*",
+				"System.Linq.Expressions": "4.1.0-*",
```

Comment:
I'd prefer not to take an additional (and typically unnecessary) dependency in this library. As above, I think this should move up to Pomelo.EF.